schema_utils_test.py

```diff
@@ -546,13 +546,15 @@ def check_normalization(self, schema, mappings, invalid_items):
         """Validates the schema and tests that values are normalized correctly.
 
         Args:
-            schema: dict. the schema to normalize the value against. Each schema
-                is a dict with at least a key called 'type'. The 'type' can take
-                one of the SCHEMA_TYPE_* values declared above.
-            mappings: list. a list of 2-element tuples. The first element of
-                each item is expected to be normalized to the second.
-            invalid_items: list. a list of values. Each of these is expected to
-                raise an AssertionError when normalized.
+            schema: dict. The schema to normalize the value
+                against. Each schema is a dict with at least a key called
+                'type'. The 'type' can take one of the SCHEMA_TYPE_* values
+                declared above.
+            mappings: list(tuple(double, double)). A list of 2-element tuples.
```

Comment:
Python doesn't use "double". Please fix. Also, please explain how you know it's a double.